code/modules/reagents/reagent_containers/food/drinks/drinkingglass.dm

```diff
@@ -583,6 +630,11 @@
 			var/mob/living/carbon/M = loc
 			M.update_inv_hands()
 
+/obj/item/weapon/reagent_containers/food/drinks/drinkingglass/examine(mob/user)
+	..()
+	if(reagents.get_master_reagent_id() == METABUDDY && user.client)
```

Comment:
due to questionable decisions, `user` may not be of type `mob`. You need an `istype(user)` check before you access `.client`. see #17400